python_modules/dagster/dagster/cli/run.py

```diff
@@ -11,9 +12,17 @@ def create_run_cli_group():
 
 
 @click.command(name="list", help="List the runs in this dagster installation.")
-def run_list_command():
+@click.option(
+    "--limit", help="Only list a specified number of instances", is_flag=True, type=int,
```

Comment:
Nit: `number of runs` instead of `number of instances`. I also don't think you need to have `is_flag` here, because this isn't a boolean flag arg.